src/sagas/PostSaga.js

```diff
@@ -0,0 +1,19 @@
+import { put, call} from 'redux-saga/effects';
+
+import {getPostDataSuccess } from '../actions';
+import axios from "axios";
+import { urlGetPost } from '../api';
+
+function fetchPost() {
```

Comment:
```suggestion
export default function getPostsList()
```
Nhưng function này bỏ vào API chứ không bỏ trong saga em nha, bên ni em chỉ import vào gọi thôi